README.md

```diff
@@ -44,14 +44,14 @@ See the [How to Contribute page][9].
 
 [1]: https://github.com/DataDog/datadog-agent/
 [2]: https://github.com/DataDog/extendeddaemonset
-[3]: https://github.com/helm/charts/tree/main/stable/datadog
+[3]: https://github.com/DataDog/helm-charts
```

Comment:
Should this point to the datadog helm chart instead of the helm-charts repo?
https://github.com/DataDog/helm-charts/tree/main/charts/datadog

Reply:
To this for example ? https://github.com/DataDog/helm-charts/tree/main/charts/datadog-operator

Reply:
The updated link (`[3]`) appears in this sentence: `The official [Datadog Helm chart][3] is still the recommended way to setup Datadog in a Kubernetes cluster`.

The helm-charts contains several charts (agent, EDS, operator, etc.). That's why I think we should link to the specific chart the sentence is referring to which is https://github.com/DataDog/helm-charts/tree/main/charts/datadog

Reply:
Oh yeah you are right the sentence is actually referring to this one. Will update the link, thanks !